CONTRIBUTING.adoc

```diff
@@ -11,74 +18,81 @@ The best way to install Node is to use {uri-nvm}[nvm] (Node Version Manager).
 
 We recommend using the latest long term support (LTS) release of Node. While you can use other versions of Node, this extension is only tested against active LTS releases.
 
+.Note
```

Comment:
Why are we adding a title when the NOTE admonition already has a label? To me, this is adding AsciiDoc for presentation, which should not be done.